common/services/prismic/articles.js

```diff
@@ -361,8 +361,12 @@ function parseContentLink(document: ?PrismicDocument): ?MultiContent {
 
 function parseArticleDoc(document: PrismicDocument): Article {
   const { data } = document;
+  // When we imported data into Prismic from the Wordpress blog some content
+  // needed to have it's original publication date displayed. This is no
+  // longer a feature that editors can, but we still want to display the value
```

Comment:
nit: "its original punctuation date", not "it's". no possessive apostrophe.

"this is no longer a feature that editors can" <~ can what?

Reply:
Made the comment simpler.